diode-proto/diode/v1/reconciler.proto

```diff
@@ -111,10 +111,38 @@ message RetrieveIngestionLogsResponse {
   string next_page_token = 3; // Token for the next page of results, if any
 }
 
+// The request to take action on an ingestion log
+message ActionIngestionLogRequest {
+  message DiffAction {}
+  message ApplyAction {}
+  message RejectAction {}
+
+  oneof action {
+    DiffAction diff = 1;
+    ApplyAction apply = 2;
+    RejectAction reject = 3;
+  }
+  string ingestion_log_id = 4;
+  optional string branch_id = 5;
+}
+
+// The response from the ActionIngestionLog request
+message ActionIngestionLogResponse {
+  message Error {
+    string message = 1;
+    int32 code = 2;
+  }
```

Comment:
The IngestionLog returned has its own set of errors, not sure what the shape of these errors is currently so I just tacked this in ...

Reply:
> A little copying is better than a little dependency